src/BehaviorsSDKNative/Microsoft.Xaml.Interactions.DesignTools/Microsoft.Xaml.Interactions.DesignTools.csproj

```diff
@@ -0,0 +1,79 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{C19EE6E7-244F-4E2F-B55B-91E5C266FC38}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.Xaml.Interactions.DesignTools</RootNamespace>
+    <AssemblyName>Microsoft.Xaml.Interactions.DesignTools</AssemblyName>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <Deterministic>true</Deterministic>
+	<OutputPath>..\..\..\out\$(SolutionName)\bin\$(Platform)\$(Configuration)\</OutputPath>
+    <IntermediateOutputPath>..\..\..\out\$(SolutionName)\obj\$(Platform)\$(Configuration)\</IntermediateOutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);SurfaceIsolation</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.DesignTools.Extensibility, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.VisualStudio.DesignTools.Extensibility.16.4.29519.181\lib\net45\Microsoft.VisualStudio.DesignTools.Extensibility.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.VisualStudio.DesignTools.Interaction, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.VisualStudio.DesignTools.Extensibility.16.4.29519.181\lib\net45\Microsoft.VisualStudio.DesignTools.Interaction.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
```

Comment:
How about we replace this with a NuGet PackageReference for Microsoft.VisualStudio.DesignTools.Extensibility now?
See [https://www.nuget.org/packages/Microsoft.VisualStudio.DesignTools.Extensibility/](https://www.nuget.org/packages/Microsoft.VisualStudio.DesignTools.Extensibility/)

Reply:
Hah, just saw the packages.config. Can/should we switch to package reference though?

---
In reply to: [382680072](https://github.com/microsoft/XamlBehaviors/pull/172#discussion_r382680072) [](ancestors = 382680072)

Reply:
It's a NuGet package reference. How else should we reference it?

Reply:
Using <PackageReference .../> instead of packages.config. See the change you made to NativeUnitTests.csproj

---
In reply to: [382847354](https://github.com/microsoft/XamlBehaviors/pull/172#discussion_r382847354) [](ancestors = 382847354)

Reply:
Thanks for the clarification. I migrated all applicable projects from packages.config to PackageReference.